src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ CHANGELOG
    `model_timezone` option in `DateType`, `DateTimeType`, and `TimeType`
  * Deprecate `PostSetDataEvent::setData()`, use `PreSetDataEvent::setData()` instead
  * Deprecate `PostSubmitEvent::setData()`, use `PreSubmitDataEvent::setData()` or `SubmitDataEvent::setData()` instead
+ * Use `form.post_set_data` instead of `form.pre_set_data` in `ResizeFormListener`
+ * Change the priority of `DataCollectorListener` from 255 to -255
```

Comment:
Are we sure about this? From experience this can lead to unwanted behaviors, eg what if a listener stop propagation? (is that possible with Doctrine?)
What's the motivation for this change of priority? Is that just modern-us thinking ancient-us made a mistake? Given the comment, this looks on purpose to me.

Reply:
It raises the same concern as @xabbuh in https://github.com/symfony/symfony/pull/51041#discussion_r1270524066.
Yes, I do think that was a mistake.
Given than we don't currently use post set data event in core, I cannot think of a legit use case where propagation is stopped.
To ensure that this listener is called, we could use a flag and `__destruct()` to warn or deprecate such behavior.

Reply:
OK (no need for the destruct check)